html/pages/settings/webui.inc.php

```diff
@@ -18,13 +18,21 @@
     ),
 );
 
+$dashboard_conf = array(
+    array('name'               => 'webui.default_dashboard_id',
+          'descr'              => 'Set global default dashboard id',
+          'type'               => 'text',
```

Comment:
Probably a select type would be better here.  You could modify generate_dynamic_config_panel() to output a display name "username:dashboard"